docs/examples/jaeger-docker.md

```diff
@@ -0,0 +1,117 @@
+# Collecting traces using Jaeger
+
+In this example, we are going to use [Jaeger](https://jaegertracing.io/) to collect and visualize traces produced by an
+application.
+We will cover the installation and configuration of Jaeger, as well as the instrumentation of the application using the
+otel4s library.
+
+### Project setup
+
+Configure the project in the `build.sbt`:
+
+```scala
+libraryDependencies ++= Seq(
+  "org.typelevel" %% "otel4s-java-auto" % "@VERSION@", // <1>
+  "io.opentelemetry" % "opentelemetry-exporter-otlp" % "@OPEN_TELEMETRY_VERSION@" // <2>
+)
+run / fork := true
+javaOptions += "-Dotel.java.global-autoconfigure.enabled=true" // <3>
+javaOptions += "-Dotel.service.name=jaeger-example"            // <4>
+javaOptions += "-Dotel.metrics.exporter=none"                  // <5>
+```
+
+1) Add `otel4s` library  
+2) Add an OpenTelemetry exporter. Without the exporter, the application will crash  
+3) Enable OpenTelemetry SDK autoconfigure mode  
+4) The name of the application to use in the traces  
```

Comment:
```suggestion
4) Add the the name of the application to use in the traces
```
Same as the other just for readability / flow

Reply:
Thanks, I've updated both cases.